src/llmq/quorums.cpp

```diff
@@ -526,14 +560,12 @@ std::vector<CQuorumCPtr> CQuorumManager::ScanQuorums(Consensus::LLMQType llmqTyp
                 pIndexScanCommitments = vecResultQuorums.back()->m_quorum_base_block_index->pprev;
             }
         } else {
-            // If there is nothing in cache request at least cache.max_size() because this gets cached then later
-            nScanCommitments = std::max(nCountRequested, cache.max_size());
+            // If there is nothing in cache request at least keepOldConnections because this gets cached then later
+            nScanCommitments = std::max(nCountRequested, static_cast<size_t>(llmq_params_opt->keepOldConnections));
```

Comment:
nit; not sure if this would compile; if not then replace {} with ()? 
```suggestion
            nScanCommitments = std::max(nCountRequested, size_t{llmq_params_opt->keepOldConnections});
```

Reply:
for integer types imo, static_cast are probably overly verbose and don't provide any benefit over function casts
if type will be changed from int to int* , by cast expression you won't catch a bug, but static_cast<int>() immediately show you are mistake. It is just more error-prune

For any type convert static/const/dynamic/reintrrpret_cast better than {} or C-style.
Please keep static_cast here